otterdog/eclipse-edc.jsonnet

```diff
@@ -457,6 +457,29 @@ orgs.newOrg('eclipse-edc') {
         },
       ],
     },
+    orgs.newRepo('Technology-HuaweiCloud') {
+      allow_rebase_merge: false,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      has_discussions: true,
+      has_wiki: false,
+      squash_merge_commit_title: "PR_TITLE",
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
+      },
+      secrets: [
+        orgs.newRepoSecret('DISCORD_WEBHOOK_GITHUB') {
+          value: "********",
+        },
+        orgs.newRepoSecret('EF_JENKINS_TOKEN') {
+          value: "********",
+        },
+        orgs.newRepoSecret('EF_JENKINS_USER') {
+          value: "********",
+        },
```

Comment:
@mspiekermann this requires more context. What are you trying to do with the EF_JENKINS secrets?

Reply:
As with the other Technology-XYZ repositories, they are part of the release process and thus during the workflow need those secrets.

Reply:
could this be outdated? I checked where the EF_JENKINS_TOKEN is being used, and the only reference is here:

https://github.com/eclipse-edc/.github/blob/86b7c9cc416398c6f497141bf69ad4265361a247/.github/workflows/_trigger-snapshot.yml#L14

this does not seem to use the passes jenkins_token at all.

There was a commit some time ago that added publishing directly from GitHub Actions:

https://github.com/eclipse-edc/.github/commit/ec430c3470c5bf7efd2976f685b6d88cad56ef90

maybe in this refactoring it was forgotten to cleanup unused secrets?

Reply:
Will check

Reply:
ok I removed them from the new repo already as it does not make sense to add it first and then remove again.